src/app/menus/navbar/navbar.component.html

```diff
@@ -1,30 +1,6 @@
-<div class="sample-wrapper">
-        <section class="sample-content">
-            <article class="sample-column">
-                <h4 class="sample-title">With nav and action icons</h4>
-                <div class="navbar-sample">
-                    <igx-navbar title="Sample App" actionButtonIcon="menu">
-                        <igx-icon>search</igx-icon>
-                        <igx-icon>favorite</igx-icon>
-                        <igx-icon>more_vert</igx-icon>
-                    </igx-navbar>
-                </div>
-            </article>
-
-            <article class="sample-column">
-                <h4 class="sample-title">With nav icon</h4>
-                <div class="navbar-sample">
-                    <igx-navbar [title]="currentView" actionButtonIcon="arrow_back" [isActionButtonVisible]="canGoBack()" (onAction)="navigateBack()">
-                    </igx-navbar>
-                </div>
-            </article>
-
-            <article class="sample-column">
-                <h4 class="sample-title">Without nav icon</h4>
-                <div class="navbar-sample">
-                    <igx-navbar [title]="currentView" [isActionButtonVisible]="false">
-                    </igx-navbar>
-                </div>
-            </article>
-        </section>
-    </div>
\ No newline at end of file
+<article class="sample-column">
```

Comment:
Same here.

Reply:
This class brings the margin of the navbar so I will leave it if you don't mind. However, I changed it to div and removed the following one.